src/tool/subcommands/snapshot_cmd.rs

```diff
@@ -59,6 +76,9 @@ pub enum SnapshotCommands {
         /// Path to a snapshot CAR, which may be zstd compressed
         #[arg(required = true)]
         snapshot_files: Vec<PathBuf>,
+        /// Fail at the first invalid snapshot
+        #[arg(long)]
+        fail_fast: bool,
```

Comment:
nit: I reckon this has to be a required argument, it's best not to assume what the use-case is, especially if it means a potentially very long runtime.

Reply:
Do you prefer this `#[arg(long, required = true, action = clap::ArgAction::Set)]`

Reply:
Isn't `long, required = true` enough?

Reply:
in fact - feel free to disregard this. Let's assume the default behaviour is to validate all the snapshots, like it is now.